DTS-STN/Alpha-Site
# fix docker builds

# Description

Today, an issue arose where when trying to run a new build in the prod pipeline. The container build goes succesfully, but when accessing the page, the application throws `Error: Cannot find module 'prop-types'` and a 500 error. I am not sure why this didn't seem to be an issue during the last release last week as it is exactly the same code, but through testing it appears to be a problem with the package-lock.json being broken and how dependencies are being installed while building the container.

I reproduced the error with a local `docker build .` and have mitigated it by having docker install the npm dependencies the way they are installed in npm v6 (our containers are using v7), and by re-building the package-lock.json with the same flag.

## Acceptance Criteria

docker build and run should work.

## Test Instructions

1. `docker build .`
2. run container
3. hit the site at the container's port
4. see that it works

## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
